README.md

```diff
@@ -42,6 +42,16 @@ The `vrt.sh` script is meant to be called regularly via a Cronjob (every 5 hours
 or more, as the datasets will be downloaded and published again at each run),
 with the previous variables defined.
 
+## Optional variables
+
+The following environment variables are optional:
+
+```
+# export ACTIVATE_GEOMETRY_TYPE=false 
```

Comment:
I think the variable name needs to be more explicit

Reply:
Oh, I just understood one can now define the geometry type for the created layer
IMO we do not need such generalization, because generally speaking one does not know which geometry type will be provided as inputs in vrt files submitted.

So, I would just intrioduce a var named "PROMOTE_TO_MULTI" to enable "-nlt PROMOTE_TO_MULTI"